packages/diagrams/backend/sirius-components-diagrams-layout/src/test/java/org/eclipse/sirius/components/diagrams/layout/services/DefaultTestDiagramDescriptionProvider.java

```diff
@@ -285,11 +285,15 @@ private Function<VariableManager, List<org.eclipse.sirius.components.representat
     }
 
     public DiagramDescription getDefaultDiagramDescription(Diagram diagram) {
+        return this.getDefaultDiagramDescription(diagram, false);
+    }
+
+    public DiagramDescription getDefaultDiagramDescription(Diagram diagram, boolean isCollapsible) {
```

Comment:
Do not specify such low level details here. You can simply clone a new NodeDescription. You don't need to modify that